library/src/main/java/com/pokegoapi/api/map/fort/Pokestop.java

```diff
@@ -261,20 +260,43 @@ public boolean hasLurePokemon() {
 	}
 
 	/**
+	 * Returns whether this pokestop has an active lure when detected on map.
+	 *
+	 * @return lure status
+	 */
+	public boolean hasLure() {
+		try {
+			return hasLure(false);
+		} catch (LoginFailedException e) {
```

Comment:
Mine is fine! I just red one times when i tried to catch some Exceptions on a previous PR. So it will be fine if I chain them?